katas/content/distinguishing_unitaries/index.md

```diff
@@ -63,6 +63,66 @@ This kata offers you a series of tasks in which you are given one unitary from t
     ]
 })
 
+@[exercise]({
+    "id": "distinguishing_unitaries__z_mz",
+    "title": "Z or -Z?",
+    "path": "./z_mz/",
+    "qsDependencies": [
+        "../KatasLibrary.qs",
+        "./Common.qs"
+    ]
+})
+
+@[exercise]({
+    "id": "distinguishing_unitaries__rz_r1",
+    "title": "Rz or R1?",
+    "path": "./rz_r1/",
+    "qsDependencies": [
+        "../KatasLibrary.qs",
+        "./Common.qs"
+    ]
+})
+
+@[exercise]({
+    "id": "distinguishing_unitaries__y_xz",
+    "title": "Y or XZ?",
+    "path": "./y_xz/",
+    "qsDependencies": [
+        "../KatasLibrary.qs",
+        "./Common.qs"
+    ]
+})
+
+@[exercise]({
+    "id": "distinguishing_unitaries__y_xz_my_mxz",
+    "title": "Y or XZ or -Y or -XZ?",
+    "path": "./y_xz_my_mxz/",
+    "qsDependencies": [
+        "../KatasLibrary.qs",
+        "./Common.qs"
+    ]
+})
+
+@[exercise]({
+    "id": "distinguishing_unitaries__rz_ry",
+    "title": "Rz or Ry?",
+    "path": "./rz_ry/",
+    "qsDependencies": [
+        "../KatasLibrary.qs",
+        "./Common.qs"
+    ]
+})
+
+@[exercise]({
```

Comment:
Looking at this with a fresh eye, I'm wondering if this is the best order of exercises - the difference between the probabilistic approach in DistinguishRzFromRy and the clever but much simpler trick here is quite noticeable. Do you think it would make sense to move this task earlier, to right before DistinguishRzFromRy?

Reply:
you're right in that the rz/ry one is where it starts to get a bit more complex so this change in kata progression order makes sense.